wallet/wallet-rpc-lib/src/rpc/mod.rs

```diff
@@ -351,6 +351,29 @@ impl<N: NodeInterface + Clone + Send + Sync + 'static> WalletRpc<N> {
             .await?
     }
 
+    pub async fn confirmed_transactions(
```

Comment:
The word "confirmed" is very dangerous to use. We still haven't formalized what "confirmed" really means. In bitcoin, confirmation depends on the number of blocks stacked above the block where the transaction resides. You'll notice also that "confirmed" always comes accompanied with a parameter "depth" or "number of confirmation", as a filter, exactly for this reason.

For us, it's much more complicated and there's a fair amount of math involved to decide what a "confirmed" transaction really is.

I would feel more comfortable calling this `mainchain_transactions`, because that doesn't give any sense of "confirmed". We can go back to the problem of "confirmations" later.